Earthfile

```diff
@@ -12,11 +12,16 @@ ENV RUST_VERSION=1.70.0
 ENV RUST_BUILD_MODE='' # set to --release for release builds
 
 install-deps:
-    RUN apt-get update
+    RUN apt-get update --fix-missing
```

Comment:
Why is --fix-missing required?

Reply:
Good question `apt-get install locale` fails without this.